vita3k/modules/SceDriverUser/SceErrorUser.cpp

```diff
@@ -17,8 +17,13 @@
 
 #include <module/module.h>
 
-EXPORT(int, sceErrorGetExternalString) {
-    return UNIMPLEMENTED();
+#include <util/tracy.h>
+TRACY_MODULE_NAME(SceErrorUser);
+
+EXPORT(SceInt32, sceErrorGetExternalString, char *result, int err) {
+    TRACY_FUNC(sceErrorGetExternalString, result, err);
+    sprintf(result, "0x%08X", err);
+    return 0;
```

Comment:
Turn it into a `CALL_EXPORT(_sceErrorGetExternalString)`

Reply:
Oh... i have thinks is useless because code is very small... a’d so no wanted create include etc for ridiculus things, is juste one line, no needed

Reply:
So is _sceKernelCancelEventFlag, _sceKernelCancelEventFlag, _sceKernelCancelSema... Yet sceKernelCancelEventFlag, sceKernelCancelEventFlag or sceKernelCancelSema still use CALL_EXPORT in order not to have to modify code at two locations if it ever happens or forget to modify one of the two implementations.

Reply:
Yes but it have already one include for declare function, i have thinks is not needed here for now

Reply:
sorry for delay, is done